frontend/src/App.jsx

```diff
@@ -1,22 +1,58 @@
-import './App.css'
 import HomePage from "./components/homePage/HomePage.jsx"
```

Comment:
If possible, let's keep frontend and backend changes as separate commits or PRs. So when you're developing, you are mostly likely making both backend and frontend changes. So when creating the commit, you can selectively add backend changes into the first commit and frontend changes into the second commit since it is most likely that your frontend changes is dependent on your backend changes.

Reply:
Done